src/Tizen.Wearable.CircularUI.Chart.Forms/BarChartView.cs

```diff
@@ -0,0 +1,155 @@
+    {
+        /// <summary>
+        /// BindableProperty. Identifies the BarChartType bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty BarChartTypeProperty = BindableProperty.Create(nameof(BarChartType), typeof(BarChartType), typeof(BarChartView), BarChartType.Vertical);
+
+        /// <summary>
+        /// BindableProperty. Identifies the BarWidth bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty BarWidthProperty = BindableProperty.Create(nameof(BarWidth), typeof(double), typeof(BarChartView), 10d);
+
+        /// <summary>
+        /// BindableProperty. Identifies the BarBackgroundColorIsVisible bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty BarBackgroundColorIsVisibleProperty = BindableProperty.Create(nameof(BarBackgroundColorIsVisible), typeof(bool), typeof(BarChartView), false);
+
+        /// <summary>
+        /// BindableProperty. Identifies the BarTopRadius bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty BarTopRadiusProperty = BindableProperty.Create(nameof(BarTopRadius), typeof(double), typeof(BarChartView), 0d);
+
+        /// <summary>
+        /// BindableProperty. Identifies the BarBottomRadius bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty BarBottomRadiusProperty = BindableProperty.Create(nameof(BarBottomRadius), typeof(double), typeof(BarChartView), 0d);
+
+        /// <summary>
+        /// BindableProperty. Identifies the AxisOption bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty AxisOptionProperty = BindableProperty.Create(nameof(AxisOption), typeof(AxisOption), typeof(BarChartView), default(AxisOption));
+
+        /// <summary>
+        /// BindableProperty. Identifies the ValueLabelPosition bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty ValueLabelPositionProperty = BindableProperty.Create(nameof(ValueLabelPosition), typeof(ValueLabelPosition), typeof(BarChartView), ValueLabelPosition.End);
+
+        /// <summary>
+        /// BindableProperty. Identifies the ValueLabelIsVisible bindable property.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public static readonly BindableProperty ValueLabelIsVisibleProperty = BindableProperty.Create(nameof(ValueLabelIsVisible), typeof(bool), typeof(BarChartView), false);
+
+        /// <summary>
+        /// Gets or sets type of bar chart.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public BarChartType BarChartType
```

Comment:
I think, it should be a member of `AxisOption` and Name is not proper as I understand, I think it named `MajorAxis`